elichika/tests/syntax/Continue.py

```diff
@@ -0,0 +1,58 @@
+# coding: utf-8
+
+import chainer
+import chainer.functions as F
+
+
+class Continue(chainer.Chain):
+    def forward(self, x, y):
+        ret = 0
+        for i in range(x):
+            if i == y:
+                continue
+            ret = ret + i
+        return ret
+
+
+class ContinueNested(chainer.Chain):
+    def forward(self, x, y):
+        ret = 0
+        for i in range(x):
+            for j in range(x):
+                if i == y:
+                    if j == y:
+                        continue
+                ret = ret + j
+        return ret
+
+
+class ContinueNestedMultiple(chainer.Chain):
+    def forward(self, x, y):
+        ret = 0
+        for i in range(x):
+            if i == y - 1:
+                continue
+            for j in range(x):
+                if i == y:
+                    if j == y:
+                        continue
+                ret = ret + j
+        return ret
+
+
```

Comment:
It seems `len(self.for_continue_stack[-1]) > 1` is not tested? I tried to add a test case for this case but it failed due to a bug in elichika. I think the `assert canonicalize(to_ast(source)) == to_ast(target)` style test you are adding will cover the case, but it will be helpful for us to add a commented-out test like (please just use TODO(hamaji))

https://github.com/pfnet-research/chainer-compiler/blob/master/elichika/tests/syntax/Alias.py#L54

so that we won't forget the issue.

Of course, fixing the bug itself is also welcomed :)